### ISSUE TITLE
Input: supported html types

### ISSUE THREAD

**Issue by dinony** [environment-agency-austria/react-ocean-forms-bootstrap#75]
Hi,

In the [docs](https://environment-agency-austria.github.io/forms-showcase/#/api/bootstrap/input/) the type attribute is claimed to allow:

> HTML5 input type of the input element. 

In a form I have to use radio buttons (type="radio"), but I think it is not supported?

**Comment by maschino** [comment]
You're right - type radio (and type checkbox) are currently not supported. For simple binary inputs you could use [OnOffToggleButton](https://environment-agency-austria.github.io/forms-showcase/#/components/bootstrap/on-off-toggle-button/) instead.

Could you explain your use case? Maybe we can add a component to satisfy it.

**Comment by dinony** [comment]
The use case is to use radio buttons. For now the analyst agreed on to use [select](https://environment-agency-austria.github.io/forms-showcase/#/components/bootstrap/select/). So, this is not an immediate feature request. At least the docs could be updated and the type prop could be type safer. Thanks!
